### DISCUSSION TITLE
Web UI out of date when working with branches?

### DISCUSSION THREAD

**Discussion by janpio** [planetscale/discussion#55] [Q&A] [answered]
I am currently starting to discover how branches work. Unfortunately often the web UI does not reflect my schema changes I did and can see with my database UI:

![image](https://user-images.githubusercontent.com/183673/120064856-a3aea780-c06e-11eb-92cd-608ba2093440.png)

vs.

![image](https://user-images.githubusercontent.com/183673/120064865-b32df080-c06e-11eb-97f8-13ff47a90df9.png)

(this is the first branch of this database)

Am I doing something wrong, is it expected that this is a few minutes out of sync, or is the system overloaded/slow right now?

PS: The same is true for the branch listing, I suspect this is related:

![image](https://user-images.githubusercontent.com/183673/120064899-e83a4300-c06e-11eb-925f-cbcdf03aeb20.png)
(Screenshot shows an updated state though)

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @janpio, 

Did you try clicking <kbd>Refresh schema</kbd> on that page?

Currently you'll need to click that to see your latest schema. We'll be making this automatic in the future.

**Reply by janpio** [reply]
I think I did click that button. 
I know I definitely refreshed the list of branches.

Will hopefully remember this issue next time I encounter this and report back again.

**Reply by mscoutermarsh** [reply]
Hi @janpio, found a bug yesterday that might explain the problem you were seeing. We got it fixed.

Basically clicking that button occasionally had a long delay, which made it seem broken. Thanks again!
